snappiershot/snapshot/metadata.py

```diff
@@ -1,11 +1,48 @@
 """ Metadata object used to identity and track snapshots. """
-from typing import Dict
+from typing import Any, Dict
+
+from numpy import all, ndarray
```

Comment:
Style question - is it preferred to 'import numpy as np' and then call np.all and np.ndarray later?

Reply:
i was under the impression that if you dont need all of numpy, it's good to just import what exactly you need. idk though tbh

Reply:
It is better practice to only import what you need. Also saves memory